exporter/prometheusremotewriteexporter/factory.go

```diff
@@ -97,6 +97,9 @@ func createDefaultConfig() component.Config {
 		TargetInfo: &TargetInfo{
 			Enabled: true,
 		},
+		ScopeInfo: &ScopeInfo{
+			Enabled: true,
```

Comment:
I don't think this should default to `true`.  We can perhaps revisit this at a later date, but I would be more comfortable starting from a position that doesn't alter existing systems by default.

Reply:
I think this makes sense.